doc/languages-frameworks/go.section.md

```diff
@@ -112,16 +112,6 @@ done
 
 Both `buildGoModule` and `buildGoPackage` can be tweaked to behave slightly differently, if the following attributes are used:
 
-### `buildFlagsArray` and `buildFlags`: {#ex-goBuildFlags-noarray}
```

Comment:
How do we want to deprecate it?

Reply:
I'll deal with this in a follow up PR.

Reply:
https://github.com/NixOS/nixpkgs/pull/132845